plugins/com.google.cloud.tools.eclipse.appengine.flex/META-INF/MANIFEST.MF

```diff
@@ -7,5 +7,9 @@ Bundle-Version: 0.1.0.qualifier
 Bundle-Vendor: Google Inc.
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-Localization: plugin
-Require-Bundle: org.eclipse.wst.common.project.facet.core
-Import-Package: com.google.cloud.tools.eclipse.appengine.facets
+Require-Bundle: org.eclipse.core.expressions,
```

Comment:
Could any of these be Import-Packages instead of Require-Bundle? The recommendation in Eclipse land is to prefer  Import-Packages where possible unless split packages or some such make Require-Bundle mandatory.